components/mdx/ComponentStyle.js

```diff
@@ -1,3 +1,4 @@
+import Li  from "./Li";
```

Comment:
This is correct but doesn't follow the same style as the other imports

```suggestion
import Li  from "@components/mdx/Li";
```